src/pycram/__init__.py

```diff
@@ -20,7 +20,7 @@
 import logging
 import logging.config
 import pycram.process_modules.available_process_modules
-
+import pycram.robot_descriptions.robot_description_handler
```

Comment:
This can be removed since the robot description needs to be explicitly imported if you want to use it.